### DISCUSSION TITLE
BOTW gyro not working in myahm agana shrine dungeon

### DISCUSSION THREAD

**Discussion by mjhmiller** [kmicki/SteamDeckGyroDSU#31] [Q&A] [answered]
so gyro works but when specifically entering a dungeon, myahm agana shrine dungeon, you will find that when you have to operate a floor using the gyro control, controls stop working

**Comment by kmicki** [comment, marked as answer]
I have just checked and [it works properly](https://imgur.com/a/qj9Xket).

1. Follow [instructions on readme page](https://github.com/kmicki/SteamDeckGyroDSU#reporting-problems) to provide logs.
2. What version of Cemu?
3. How do you use sdgryodsu? Via cemuhook or using built-in Cemu functionality?
4. How did you confirm that gyro works outside of the shrine?

**Reply by mjhmiller** [reply]
Thank you, finally even after having installed cemuhook, and using emudecks own gyro install only to find gyro works half way this finally did the trick. for what it is worth the controller profile i use has an enable gyro button and this turns on gyro from start, or just pulling out you bow will allow gyro to work just for aiming, but when in the shrine trying to manipulate the ball andmaze mini gyro game all that happened was the entire structure flipped upside down. i also hit the calibrate button after installing it through konsole. I am finally pleased to say this works fine. i will update any changes through game

**Reply by Joscbm** [reply]
Good morning, what did you do to make it work? I am having the same problem as you.

**Comment by mjhmiller** [comment]
I just followed the directions from that same GitHub page basically have to
go into the council command and be like copy the download and when it does
it that way it seems to work properly
